builder_test.go

```diff
@@ -2,6 +2,7 @@ package lifecycle_test
 
 import (
 	"bytes"
+	"errors"
```

Comment:
I think I accidentally changed this from `github.com/pkg/errors` - I can change it back.